app/Console/Commands/AdsharesCrawlCommand.php

```diff
@@ -78,7 +86,7 @@ protected function readBroadcasts(Esc $esc)
                 $this->info("Found $host -> {$log['address']}");
                 // TODO: extract algo
                 if (preg_match('/^([a-z0-9][a-z0-9-]{0,62}\.)+([a-z]{2,})$/i', $host)) {
-                    NetworkHost::registerHost(Esc::normalizeAddress($log['address']), $host);
+                    NetworkHost::registerHost(AdsUtils::normalizeAddress($log['address']), $host);
                 // TODO: check this with Jacek in adserver symfony code
                     // $nHost->setAccountMsid($log['account_msid']);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)